content/60.methods.md

```diff
@@ -123,24 +130,50 @@ Because in this implementation of Manubot, most collaborators were writing and e
 
-#### Data Analysis and Integration
+<!--To Do: Add dot plot of unique contributors & commits over time-->
 
-<!--To Do: Flow chart of data integration? We could have a summary figure showing all of the external data sources that are integrated into the manuscript. We have icons for MSSM reviews, JHU data, etc. at the top. That flows into a GitHub repo, which also takes input from all the contributors. Then the output of the repo is the manuscript and other CI artifacts.-->
+In order to make the project accessible to individuals from a number of backgrounds, we developed resources explaining how to use GitHub's web interface to develop and edit text and interact with Manubot for individuals with no prior experience working with git or other version control platforms.
+We developed tutorials containing visuals to explain how to open an issue, open a pull request, and review a pull request [@url:https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md; @url:https://github.com/greenelab/covid19-review/blob/master/INSTRUCTIONS.md].
+Additionally, the framework for evaluating literature was converted into issue templates to simplify review of new articles.
+Articles were classified as _diagnostic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-diag-study-template.md], _therapeutic_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-therapy-study-template.md], or _other_ [@url:https://github.com/greenelab/covid19-review/blob/master/.github/ISSUE_TEMPLATE/new-paper-template.md], with an associated template developed to guide the review of papers and preprints in each category.
+<!-- To Do: Number of paper issue and number of unique contributors?-->
+
+Currently, three manuscripts associated with this project have been submitted to _mSystems_, with one accepted for publication [@arxiv:2102.02250] and two under revision [@arxiv:2103.02723; @arxiv:2102.01521].
+An additional four manuscripts are in preparation.
+All seven will be submitted to _mSystems_ as part of a special issue that is providing support for evolving reviews, so that they will continue to be updated as more information becomes available.
+These manuscripts cover a wide range of topics including the fundamental biology of SARS-CoV-2 (pathogenesis [@arxiv:2102.01521] and evolution [@individual-evolution]), biomedical advances in responding to the virus and COVID-19 (pharmaceutical therapeutics [@arxiv:2103.02723], nutraceutical therapeutics [@arxiv:2102.02250], vaccines [@individual-vaccines], and diagnostic technologies [@individual-diagnostics]), and biological and social factors influencing variable effects on patients [@individual-inequality].<!--To Do: add these additional individual citations-->
+To date, [number] authors are associated with the consortium.<!--To Do: break down by undergrad, grad/med student, post-doc, jr faculty?-->
```

Comment:
This could a place to cross-reference growth in the number of authors if we plot that.